portfolio.html

```diff
@@ -0,0 +1,325 @@
+                don’t hestiate to contact me.</p>
+        </div>
+
+
+        <div class="social-media">
+            <ul>
+                <li> 
+                    <a href="#"><img class="social" src="socialmedia/github.png" alt="github symbol"></a>
+              </li>
+                <li>
+                    <a href="#"><img class="social" src="socialmedia/linkidin.png" alt="linkidin symbol"></a>
+                  </li>
+                    
+                <li><a href="#"><img class="social" src="socialmedia/tump.png" alt="hand-space-o symbol"></a>
+                </li>
+                    
+                <li><a href="#"><img class="social" src="socialmedia/tiwtter.png" alt="tiwtter symbol"></a>
+                    </li>
+                <li><a href="#"><img class="social" src="socialmedia/m.png" alt="Icon-medium"></a>
+                    </li>
+            </ul>
+            
+        </div>
+
+
+
+    </div>
+    <div class="group">
+        <img id="vector" src="images/Group.png" alt="group of white vector images">
+    </div>
+</div>
+</section>
+
+
+<section class="section2">
+    <div class="myRecent">
+        <div class="works">
+            <h1 class="recent">My Recent Works</h1>
+            <img class="vector1" src="images/Vector.png" alt="under line as an image">
+            
+            <img class="image1" src="images/Placeholder.png" alt="place holder image">
+            <h2 class="multi">Multi-Post Stories</h2>
+            <p class="selection">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+    
+    <div class="buttons">
+        <ul>
+           <li><button class="btn">css</button></li> 
+           <li><button class="btn">html</button></li>
+            <li><button class="btn">bootstrap</button></li>
+            <li><button class="btn">Ruby</button></li>
+        </ul>
+    </div>
+    <div class="see">
+        <a class="see1" href="#">See project</a>
+    </div>
+
+
+
+    <div class="profesional">
+        <div class="profesional_art">
+            <h2 class="art">Profesional Art Printing Data</h2>
+            <p class="daily">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="buttons1">
+            <ul>
+                <li><button class="btn1">html</button></li>
+                <li><button class="btn1">bootstrap</button></li>
+                <li><button class="btn1">Ruby</button></li>
+            </ul>   
+        </div> 
+        <div class="seep">
+            <a class="project" href="">See Project</a>
+            
+        </div>
+    
+    </div>
```

Comment:
- [ ] Kindly be noticed that the project has an image, that could use an `img` or use create as a background.
- [ ] Also you should the Figma design.

Your Design | Figma Design
---------------- | -------------------
![Screenshot from 2022-06-30 17-29-18](https://user-images.githubusercontent.com/33752735/176717117-9afbe541-7636-4d2b-a4d9-9801beef1aa0.png) | ![Screenshot from 2022-06-30 17-29-00](https://user-images.githubusercontent.com/33752735/176717152-c21ddbec-f915-4489-bc4a-7d752fe27297.png)